tests/web_test_helpers.py

```
@@ -0,0 +1,426 @@
+        "current_tag": current_tag,
+        "latest_tag": latest_tag,
+        "target_image": target_image,
+        "restart_container": restart_container,
+    }
+
+
+def _setup_admin(
+    client: TestClient,
+    *,
+    username: str = "admin",
+    password: str = "correct horse battery staple",
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to function default: "password"

(S107)